packages/tools/client-debugger/client-debugger-view/src/Debugger.tsx

```diff
@@ -40,82 +54,98 @@ export interface FluidClientDebuggersProps {
  * @remarks If no debugger has been initialized, will display a note to the user and a refresh button to search again.
  */
 export function FluidClientDebuggers(props: FluidClientDebuggersProps): React.ReactElement {
-	const debuggerRegistry: DebuggerRegistry = getDebuggerRegistry();
+	const [containers, setContainers] = React.useState<ContainerMetadata[]>([]);
+	const [menuSelection, setMenuSelection] = React.useState<string>("");
```

Comment:
Nit: Can we use `undefined` instead of empty strings / arrays here for the initial state? Especially for the container list, I think empty signifies there are no registered debuggers, while undefined signals we haven't requested the list yet.